packages/charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -141,29 +141,31 @@ export class LineChartBase extends React.Component<
   private _fitParentContainer(): void {
     const { containerWidth, containerHeight } = this.state;
     if (this.props.parentRef) {
-      setTimeout(() => {
+      requestAnimationFrame(() => {
```

Comment:
Consider canceling this `requestAnimationFrame` in `componentWillUnmount` to avoid potentially leaking memory.

>Return value
>
>A long integer value, the request id, that uniquely identifies the entry in the callback list. This is a non-zero value, but you may not make any other assumptions about its value. You can pass this value to window.cancelAnimationFrame() to cancel the refresh callback request.
>
>https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame#Return_value